src/main/resources/app_config.yml

```diff
@@ -3,16 +3,16 @@ server:
   applicationContextPath: /
   applicationConnectors:
     - type: http
-      port: 8888
+      port: 9000
 database:
   # the name of your JDBC driver
   driverClass: org.postgresql.Driver
 
   # the username
-  user: dawidkw
+  user: radoslawl
 
   # the JDBC URL
-  url: jdbc:postgresql://localhost/dawidkw
+  url: jdbc:postgresql://localhost/radoslawl
```

Comment:
We need to think of a way to stop replacing each others' configuration with every merge request